docs/runtime_options.md

```diff
@@ -167,9 +167,19 @@ in mind that it is impossible to group more events that there are counters avail
 * `uncore events` - events which can be counted by PMUs outside core.
 * `PMU` - Performance Monitoring Unit
 
+## Referenced bytes
+
+This requires to change disable_metrics from the default value so that referenced_memory is not excluded
+
+- `--referenced_read_interval` duration Read interval for referenced bytes (container_referenced_bytes metric), number of seconds after which referenced bytes are read, if set to 0 referenced bytes are never read (default: 0s)
+- `--referenced_reset_interval` duration Reset interval for referenced bytes (container_referenced_bytes metric), number of seconds after which referenced bytes are cleared, if set to 0 referenced bytes are never cleared (default: 0s)
+
+The referenced memory value is based on one of two files in `/sys/<PID>`: smaps or smaps_rollup. If the latter exists it reports the same value as the first one, but in aggregated form. This is only implementation flavoring and there is no difference whether smaps_rollup exists or not.
+*NOTE*: To avoid CPU starvation, reads are distributed over time. Keeping frequency, each reading loop gets a random offset from range <0, `referenced_read_interval`> before starting.
+
```

Comment:
Please add information about collecting referenced bytes using cAdvisor in docker container, this command works:
```
sudo docker run \
--volume=/:/rootfs:ro \
--volume=/var/run:/var/run:ro \
--volume=/sys:/sys:ro \
--volume=/var/lib/docker/:/var/lib/docker:ro \
--volume=/dev/disk/:/dev/disk:ro \
--publish=8080:8080 \
--device=/dev/kmsg \
--pid=host \
--privileged \
--name=cadvisor \
cadvisor:$CADVISOR_TAG
--disable_metrics="cpu_topology,resctrl,udp,sched,hugetlb,node_vmstat,memory_numa,tcp,advtcp,percpu,process" \
--referenced_read_interval=300s
```